benches/pub_sub.rs

```diff
@@ -4,7 +4,7 @@ use std::time::{Duration, Instant};
 use tonari_actor::{Actor, Context, Event, Recipient, System};
 
 #[derive(Debug, Clone)]
-struct StringEvent(String);
+struct StringEvent(());
```

Comment:
Was this change due to a clippy warning? If so and we just want a unit struct, then maybe just `struct ActorEvent;` is enough. A tuple struct with just the unit type seems a bit strange.

Reply:
Yeah, it's because of clippy. Will fix.